src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -1220,7 +1220,8 @@ void PaintSurface(PaintSession& session, uint8_t direction, uint16_t height, con
             {
                 // Walls
                 // Loc661089:
-                const auto fpId = static_cast<FilterPaletteID>((((mapSelectionType - 9) + rotation) & 3) + 0x21);
+                const auto fpId = static_cast<FilterPaletteID>(
+                    (((mapSelectionType - 9) + rotation) & 3) + static_cast<uint32_t>(FilterPaletteID::PaletteLandMarker));
```

Comment:
Can we perhaps introduce a constant for 9 here? same has the other line affected where it has MAP_SELECT_TYPE_QUARTER_0, that probably would make this slightly more readable then

Reply:
I assume it was done like this because MAP_SELECT_TYPE_EDGE_0 is 10 instead of 9, messing up the edge modulo.
I can change it to `(((mapSelectionType - MAP_SELECT_TYPE_EDGE_0 + 1) + rotation) & 3)`

Reply:
Its still better than not knowing what 9 means in my opinion.

Reply:
Ok, done